### PR TITLE
Styledmode support

### PR DESCRIPTION
**Pull request by praveenvitara**
Enhancement: #167

setting css for the grouped labels only if styledMode flag is set true.

styledMode can be reliably identified since HC version 7.0.0, for older versions css is always set while adding the labels.

### CODE REVIEW

File: grouped-categories.js [hunk 1]
```diff
@@ -434,9 +440,13 @@
 
 				label = chart.renderer.text(name, 0, 0, useHTML)
 					.attr(mergedAttrs)
-					.css(mergedCSS)
 					.add(axis.labelGroup);
 
+				//css should only be set for non styledMode configuration. #167
+				if (label && !isStyledMode(chart)) {
```
Comment (pawelfus) [inline grouped-categories.js:446]: There's no need to check for `Highcharts.version`, simply check if `chart.options.styledMode !== true` (old versions used to have `styledMode` undefined, so it's fine).

**Review by pawelfus — CHANGES_REQUESTED** [review]
Thanks! Just a minor thing

File: grouped-categories.js (hunk 1 above)
Comment (praveenvitara) [reply grouped-categories.js]: yes it makes sense, removed highcharts version check. please verify.

**Review by pawelfus — APPROVED** [review]
Thank you!

**Pull request merged**